app/src/main/dist/config/crate.yml

```diff
@@ -484,7 +484,7 @@
 # Therefore, you need to specify the instances you want to connect to a
 # cluster as described in the following steps:
 #
-# 1. Disable multicast discovery (enabled by default):
```

Comment:
I'd rephrase it to Enable multicast discovery and change the value below to true